### PR TITLE
Documentation: Clarify instructions about changing row_factory for SQLite3

### PR DESCRIPTION
**Pull request by SilverWingedSeraph**
When I was working through the tutorial, this was very confusing to me; so, I've added the code and clarification that would have helped me get through it faster.

### CODE REVIEW

**Comment by jeffwidman** [comment]
@SilverWingedSeraph Are you planning on updating this?

**Comment by SilverWingedSeraph** [comment]
I totally forgot about this, sorry! I'll fix it now.

File: docs/patterns/sqlite3.rst [hunk 1]
```diff
@@ -71,18 +71,36 @@ Now in each request handling function you can access `g.db` to get the
...
                     for idx, value in enumerate(row))
 
     db.row_factory = make_dicts
 
-Or even simpler::
+This will make the sqlite3 module return dicts for this database connection, which are much nicer to deal with. Even more simply, we could place this in ``get_db`` instead::
 
     db.row_factory = sqlite3.Row
 
+This would use Row objects rather than dicts to return the results of queries. These are `namedtuple` s, so we can access them either by index or by key. For example, assuming we have a ``sqlite3.Row`` called ``r`` for the rows id, FirstName, LastName, and MiddleInitial::
```
Comment (jeffwidman) [inline docs/patterns/sqlite3.rst]: I think `namedtuples` needs two `` since it's restructured text and not markdown

**Comment by SilverWingedSeraph** [comment]
I'm sorry I've been late on replying. I just graduated from high school and should be much more responsive in the future.

I fixed the formatting issue and changed the code to demonstrate some of the more useful capabilities of the Row object.

**Comment by jeffwidman** [comment]
Thanks!

**Pull request merged**